includes/ucf-social-config.php

```diff
@@ -0,0 +1,339 @@
+			$defaults = self::$option_defaults; // don't use self::get_option_defaults() here (default options haven't been set yet)
+
+			add_option( self::$option_prefix . 'include_css', $defaults['include_css'] );
+			add_option( self::$option_prefix . 'facebook_url', $defaults['facebook_url'] );
+			add_option( self::$option_prefix . 'twitter_url', $defaults['twitter_url'] );
+			add_option( self::$option_prefix . 'google_url', $defaults['google_url'] );
+			add_option( self::$option_prefix . 'linkedin_url', $defaults['linkedin_url'] );
+			add_option( self::$option_prefix . 'instagram_url', $defaults['instagram_url'] );
+			add_option( self::$option_prefix . 'youtube_url', $defaults['youtube_url'] );
+		}
+
+		/**
+		 * Deletes options via the WP Options API that are utilized by the
+		 * plugin.  Intended to be run on plugin uninstallation.
+		 *
+		 * @return void
+		 **/
+		public static function delete_options() {
+			delete_option( self::$option_prefix . 'include_css' );
+			delete_option( self::$option_prefix . 'facebook_url' );
+			delete_option( self::$option_prefix . 'twitter_url' );
+			delete_option( self::$option_prefix . 'google_url' );
+			delete_option( self::$option_prefix . 'linkedin_url' );
+			delete_option( self::$option_prefix . 'instagram_url' );
+			delete_option( self::$option_prefix . 'youtube_url' );
+		}
+
+		/**
+		 * Returns a list of default plugin options. Applies any overridden
+		 * default values set within the options page.
+		 *
+		 * @return array
+		 **/
+		public static function get_option_defaults() {
+			$defaults = self::$option_defaults;
+
+			// Apply default values configurable within the options page:
+			$configurable_defaults = array(
+				'include_css'      => get_option( self::$option_prefix . 'include_css', $defaults['include_css'] ),
+				'facebook_url'     => get_option( self::$option_prefix . 'facebook_url', $defaults['facebook_url'] ),
+				'twitter_url'      => get_option( self::$option_prefix . 'twitter_url', $defaults['twitter_url'] ),
+				'google_url'       => get_option( self::$option_prefix . 'google_url', $defaults['google_url'] ),
+				'linkedin_url'     => get_option( self::$option_prefix . 'linkedin_url', $defaults['linkedin_url'] ),
+				'instagram_url'    => get_option( self::$option_prefix . 'instagram_url', $defaults['instagram_url'] ),
+				'youtube_url'      => get_option( self::$option_prefix . 'youtube_url', $defaults['youtube_url'] ),
+			);
+
+			// Force configurable options to override $defaults, even if they are empty:
+			$defaults = array_merge( $defaults, $configurable_defaults );
+
+			return $defaults;
+		}
+
+		/**
+		 * Performs typecasting, sanitization, etc on an array of plugin options.
+		 *
+		 * @param array $list | Assoc. array of plugin options, e.g. [ 'option_name' => 'val', ... ]
+		 * @return array
+		 **/
+		public static function format_options( $list ) {
+			foreach ( $list as $key => $val ) {
+				switch ( $key ) {
+					case 'include_css':
```

Comment:
Since include_css is a boolean option, formatting the value using `filter_var( $val, FILTER_VALIDATE_BOOLEAN )` here would be beneficial.  See https://github.com/UCF/UCF-Alert-Plugin/blob/master/includes/ucf-alert-config.php#L108 for an example